src/Features/Core/Portable/Completion/Providers/AbstractInternalsVisibleToCompletionProvider.cs

```diff
@@ -148,6 +148,11 @@ private async Task AddAssemblyCompletionItemsAsync(CompletionContext context, Ca
                     properties: ImmutableDictionary.Create<string, string>().Add(ProjectGuidKey, projectGuid));
                 context.AddItem(completionItem);
             }
+
+            if (context.Items.Count > 0)
+            {
+                context.CompletionListSpan = await GetTextChangeSpanAsync(context.Document, context.CompletionListSpan, cancellationToken);
```

Comment:
needs ConfigureAwait. also wrap parameter line to make more visible.

Reply:
yes, please add ConfigureAwait

---
In reply to: [212387444](https://github.com/dotnet/roslyn/pull/29447#discussion_r212387444) [](ancestors = 212387444)